### PR TITLE
Replace AmmoPool lookup methods with properties

### PR DESCRIPTION
**Pull request by reaperrr**
And gave them more suitable names while at it.

This is more in line with how we do things in other places.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/AmmoPool.cs [hunk 1]
```diff
@@ -66,35 +66,40 @@ public class AmmoPool : INotifyCreated, INotifyAttack, IPips, ISync
 		[Sync]
 		public int RemainingTicks;
 
-		[Sync]
 		int currentAmmo;
 
+		[Sync]
+		public int CurrentAmmo
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/AmmoPool.cs]: Declare this as `{ get; private set; }` and remove `currentAmmo` completely.

Comment (reaperrr) [reply OpenRA.Mods.Common/Traits/AmmoPool.cs]: Done.

**Review by penev92 — DISMISSED** [review]
I'd rename `CurrentAmmo` to `CurrentAmmoCount` and I'd switch the two new boolean properties' order, but neither of those is a deal-breaker.
👍

**Comment by reaperrr** [comment]
Updated.

**Comment by reaperrr** [comment]
IMO it's one of those PRs that don't really need a +2, and I adressed @penev92's nits so LGTM.

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/3b43f321538d95351cd0c86e15289bb9a5af0a6c...e2aa740eb0dcbddf579b064f64af59f6b358526b

**Pull request merged**